### ISSUE TITLE
The speed of running lm_train.py in asr1 of espnet librispeech is very slow 

### ISSUE THREAD

**Issue by zdyupup** [espnet/espnet#4554]
I'm running librispeech-asr1 recipe use espnet in docker, the GPU device is 2 Tasla P100, but the speed of running lm_train.py  is very slow,
here are few logs about it, could you tell me whether the speech is right or not? and how can I improve it?

# lm_train.py --config conf/lm.yaml --ngpu 2 --backend pytorch --verbose 1 --outdir exp/train_rnnlm_pytorch_lm_unigram5000_ngpu2 --tensorboard-dir tensorboard/train_rnnlm_pytorch_lm_unigram5000_ngpu2 --train-label data/local/lm_train_unigram5000/train.txt --valid-label data/local/lm_train_unigram5000/valid.txt --resume --dict data/lang_char/train_960_unigram5000_units.txt --dump-hdf5-path data/local/lm_train_unigram5000 
# Started at Tue Aug  2 08:52:29 UTC 2022
#
2022-08-02 08:52:32,099 (lm_train:254) INFO: ngpu: 2
2022-08-02 08:52:32,099 (lm_train:257) INFO: python path = /espnet/tools/s3prl:
2022-08-02 08:52:32,104 (lm_train:274) INFO: backend = pytorch
2022-08-02 08:52:33,045 (lm:217) INFO: torch version = 1.9.0+cu111
2022-08-02 08:52:33,046 (deterministic_utils:26) INFO: torch type check is disabled
2022-08-02 08:52:34,990 (lm_utils:45) INFO: reading text dataset: data/local/lm_train_unigram5000/valid.txt

2022-08-02 09:05:13,317 (lm_utils:49) INFO: saving binary dataset: data/local/lm_train_unigram5000/train.txt.h5
2022-08-02 09:07:33,947 (lm:235) INFO: #vocab = 5002
2022-08-02 09:07:33,947 (lm:236) INFO: #sentences in the training data = 40699491
2022-08-02 09:07:33,947 (lm:237) INFO: #tokens in the training data = 1022396479
2022-08-02 09:07:33,948 (lm:240) INFO: oov rate in the training data = 0.00 %
2022-08-02 09:07:33,948 (lm:242) INFO: #sentences in the validation data = 5542
2022-08-02 09:07:33,948 (lm:243) INFO: #tokens in the validation data = 128689
2022-08-02 09:07:33,948 (lm:245) INFO: oov rate in the validation data = 0.00 %
2022-08-02 09:07:33,948 (lm:253) INFO: batch size is automatically increased (32 -> 64)
2022-08-02 09:08:14,356 (lm:268) INFO: #iterations per epoch = 767344
2022-08-02 09:08:14,356 (lm:269) INFO: #total iterations = 38367200
2022-08-02 09:08:14,404 (encoder:177) INFO: encoder self-attention layer type = self-attention
2022-08-02 09:08:18,308 (transformer:124) INFO: Tie weights set to False
2022-08-02 09:08:18,308 (transformer:125) INFO: Dropout set to 0.0
2022-08-02 09:08:18,308 (transformer:126) INFO: Emb Dropout set to 0.0
2022-08-02 09:08:18,309 (transformer:127) INFO: Att Dropout set to 0.0
2022-08-02 09:08:22,553 (lm:285) INFO: writing a model config file to exp/train_rnnlm_pytorch_lm_unigram5000_ngpu2/model.json
2022-08-02 09:08:22,568 (lm:298) WARNING: num. model params: 53,712,522 (num. trained: 53,712,522 (100.0%))

[J0           290500      3.78372     41.0679                     63174.7       
[J     total [..................................................]  0.76%
this epoch [##################................................] 37.86%
    290500 iter, 0 epoch / 50 epochs
     4.588 iters/sec. Estimated time to finish: 96 days, 1:18:54.947484.
[4A[J0           290600      3.724       40.0387                     63195         
[J     total [..................................................]  0.76%
this epoch [##################................................] 37.87%
    290600 iter, 0 epoch / 50 epochs
    4.5918 iters/sec. Estimated time to finish: 95 days, 23:24:53.089227.
[4A[J0           290700      3.68717     39.3019                     63214.7       
[J     total [..................................................]  0.76%
this epoch [##################................................] 37.88%
    290700 iter, 0 epoch / 50 epochs
     4.595 iters/sec. Estimated time to finish: 95 days, 21:48:29.133048.
[4A[J0           290800      3.71924     40.1                        63236.4       
[J     total [..................................................]  0.76%
this epoch [##################................................] 37.90%
    290800 iter, 0 epoch / 50 epochs
    4.5939 iters/sec. Estimated time to finish: 95 days, 22:21:58.568098.
[4A[J0           290900      3.74183     40.977                      63257.1       
[J     total [..................................................]  0.76%
this epoch [##################................................] 37.91%
    290900 iter, 0 epoch / 50 epochs
    4.5969 iters/sec. Estimated time to finish: 95 days, 20:51:39.737654.
[4A
